spec/unit/type/file/content_spec.rb

```diff
@@ -227,6 +227,30 @@
           end
         end
       end
+
+      SAVED_TIME = Time.now
+      [:ctime, :mtime].each do |time_stat|
+        [["older", SAVED_TIME-1, false], ["same", SAVED_TIME, true], ["newer", SAVED_TIME+1, true]].each do
+          |compare, target_time, success|
```

Comment:
Hmm, may not need a new subtask. This task makes it an error because it throws when current isn't a datetime.